cmd/wio/commands/run/dependencies/util.go

```diff
@@ -0,0 +1,11 @@
+package dependencies
+
+import (
+    "strconv"
+    "wio/cmd/wio/toolchain/npm/semver"
+)
+
+// Converts semver version to a string "Major.Minor.Patch"
+func SemverVersionToString(version *semver.Version) string {
```

Comment:
version.Str()

Reply:
Didn't realize it existed. Will use that!